apps/dotcms-ui-e2e/cypress.json

```diff
@@ -1,4 +1,10 @@
 {
+    "env": {
+        "adminUsername": "admin@dotcms.com",
+        "adminPassword": "admin",
+        "baseUrl": "http://localhost:8080"
```

Comment:
We are going to need a way to pass with URL dinamically.

Reply:
there are several ways, we should define this on our CI meeting on monday
https://docs.cypress.io/guides/guides/environment-variables.html#Setting